src/components/blocks/list/Dl/index.tsx

```diff
@@ -28,7 +28,18 @@ const StyledDl = styled.dl`
   dd {
     ${definitionListCell}
   }
-  dd > em:last-of-type {
+  dd > em:not(.italics):first-of-type {
```

Comment:
just wondering if we can separate this into a css file so we can convert it to tailwind?

Reply:
I don't think targeting children in this way with tailwind is very easy?